app/kumactl/cmd/root.go

```diff
@@ -33,19 +33,26 @@ import (
 // newRootCmd represents the base command when called without any subcommands.
 func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 	args := struct {
-		logLevel string
-		noConfig bool
+		logLevel  string
+		logFormat string
+		noConfig  bool
 	}{}
 	cmd := &cobra.Command{
 		Use:   "kumactl",
 		Short: "Management tool for Kuma",
 		Long:  `Management tool for Kuma.`,
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
-			level, err := kuma_log.ParseLogLevel(args.logLevel)
-			if err != nil {
-				return err
+			format, errFormat := kuma_log.ParseLogFormat(args.logFormat)
+			level, errLevel := kuma_log.ParseLogLevel(args.logLevel)
```

Comment:
Same about error handling.
Also as the 2 are always used together. Should we just do `format, level, err := kuma_log.ParseLogArgs(args.logFormat, args.logLevel)` this would make these less verbose

Reply:
This makes much sense. I'll do all the changes @lahabana

Reply:
> Same about error handling. Also as the 2 are always used together. Should we just do `format, level, err := kuma_log.ParseLogArgs(args.logFormat, args.logLevel)` this would make these less verbose

To achieve this we are requiring one more function in `logger.go` as `ParseLogArgs` with two other functions `ParseLogFormat` & `ParseLogLevel`. So will it be still a good idea to add @lahabana ?